pegasus/sites.v3/code.org/views/self_paced_pl/self_paced_pl_modules_carousel_elementary.haml

```diff
@@ -1,5 +1,5 @@
 :ruby
-  block_item = [
+  block_items = [
```

Comment:
I started using the plural accidentally then decided to change it everywhere 😅 I can revert if folks don't like this change